dens-validation/build.nix

```diff
@@ -35,6 +35,7 @@
 
       devShells = {
         dens-validation = hsFlake.devShell;
+        default = hsFlake.devShell;
```

Comment:
```suggestion
```

Let's remove this and leave the `default` shell for something else. The above comment (hopefully) automates the entering of the development shell.